main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        player_data = json.load(file)
```

Comment:
Consider adding error handling for the JSON loading process to manage cases where the file might not be formatted correctly or is missing.